runtime/polkadot/src/lib.rs

```diff
@@ -647,9 +648,16 @@ impl pallet_bounties::Config for Runtime {
 	type BountyValueMinimum = BountyValueMinimum;
 	type DataDepositPerByte = DataDepositPerByte;
 	type MaximumReasonLength = MaximumReasonLength;
+	type MaxActiveSubBountyCount = MaxActiveSubBountyCount;
 	type WeightInfo = weights::pallet_bounties::WeightInfo<Runtime>;
 }
 
+impl pallet_bounties::subbounty_migration::SubBountyMigration for Runtime {
+	type AccountId = AccountId;
+	type BlockNumber = BlockNumber;
+	type Balance = u128;
```

Comment:
we can use the type alias
```suggestion
	type Balance = Balance;
```